src/test/regress/expected/set_role_in_transaction.out

```diff
@@ -0,0 +1,121 @@
+-- Regression test for this issue:
+-- https://github.com/citusdata/citus/issues/3867
+SET citus.shard_count = 4;
+SET citus.next_shard_id TO 1959300;
+CREATE SCHEMA set_role_in_transaction;
+SET search_path TO set_role_in_transaction;
+CREATE TABLE t(a int);
+SELECT create_distributed_table('t', 'a');
+ create_distributed_table
+---------------------------------------------------------------------
+
+(1 row)
+
+CREATE USER user1 SUPERUSER;
+NOTICE:  not propagating CREATE ROLE/USER commands to worker nodes
```

Comment:
this notice will not show up in enterprise, can you hide it?

Reply:
Done (as well as some other output that would differ).